api/test/shell/cli_test.sh

```diff
@@ -111,12 +111,9 @@ curl -L http://localhost:2379/v3/auth/user/grant \
 # enable auth
 curl -L http://localhost:2379/v3/auth/enable -X POST -d '{}'
 
-echo "runing after etcd auth enable"
 ./manager-api &
 sleep 3
-# pkill -f manager-api
```

Comment:
@nic-chen  It's this code makes wrong. when init etcd fail, this process gets dead.  And after i try again, you are right, centos is ok.

Reply:
Congratulations, good to hear that!